src/Extension/FluentExtension.php

```diff
@@ -617,6 +620,114 @@ public function onBeforeWrite()
         }
     }
 
+    /**
+     * @throws ValidationException
+     */
+    public function onAfterWrite(): void
+    {
+        $owner = $this->owner;
+        $stage = Versioned::get_stage() ?: Versioned::DRAFT;
+        $currentLocale = FluentState::singleton()->getLocale();
+
+        if (!$this->localisedCopyActive) {
+            return;
+        }
+
+        if (!$currentLocale) {
+            return;
+        }
+
+        if (!$owner->classNameChangeDetected) {
+            // ClassName did not change so we can bail out
+            return;
+        }
+
+        if (!$owner->isInDB() || !$owner->existsInLocale()) {
+            // This is just a sanity check
+            return;
+        }
+
+        if ($stage !== Versioned::DRAFT) {
+            // Only draft stage is relevant for the duplication
+            return;
+        }
+
+        // Get list of all localised instances of this model and duplicate relations if needed (if current one has it)
+        $locales = $owner->getLocaleInstances();
+
+        /** @var Locale $locale */
+        foreach ($locales as $locale) {
+            if ($locale->Locale === $currentLocale) {
+                // We only need to handle other locale instances, current locale doesn't need updates
+                continue;
+            }
+
+            $relations = (array) $owner->config()->get('localised_copy');
```

Comment:
localised_copy is exclusively for has_one right? just checking :)

Reply:
That's correct.